src/lib/filelink.ts

```diff
@@ -1212,6 +1227,38 @@ export class Filelink {
     return this.addTask('fallback', params);
   }
 
+  /**
+   * Add zip transformation which create a zip package on files
+   * used on actual context
+   *
+   * @see https://www.filestack.com/docs/api/processing/#zip
+   * @returns this
+   * @memberof Filelink
+   */
+  zip() {
+    return this.addTask('zip', true);
+  }
+
+  /**
+   * Add task which minify css file
+   *
+   * @returns this
+   * @memberof Filelink
+   */
+  minifyCss(params: MinifyCssParams) {
+    return this.addTask('minify_css', params);
+  }
+
+  /**
+   * Add task which minify javascript file
+   *
+   * @returns this
+   * @memberof Filelink
+   */
+  minifyJs(params: MinifyJsParams) {
+    return this.addTask('minify_js', params);
```

Comment:
Add info for minify-js to use it with b64 for better targets handling, because it can produce request error